src/hotspot/cpu/ppc/globals_ppc.hpp

```diff
@@ -81,73 +81,74 @@ define_pd_global(intx, InitArrayShortSize, 9*BytesPerLong);
                    range,                                                   \
                    constraint)                                              \
                                                                             \
-  product(uintx, PowerArchitecturePPC64, 0,                                 \
+  product(uintx, PowerArchitecturePPC64, 0, DIAGNOSTIC,                     \
           "Specify the PowerPC family version in use. If not provided, "    \
           "HotSpot will determine it automatically. Host family version "   \
           "is the maximum value allowed (instructions are not emulated).")  \
                                                                             \
-  product(bool, SuperwordUseVSX, false,                                     \
-          "Use Power8 VSX instructions for superword optimization.")        \
-                                                                            \
   /* Reoptimize code-sequences of calls at runtime, e.g. replace an */      \
   /* indirect call by a direct call.                                */      \
-  product(bool, ReoptimizeCallSequences, true,                              \
+  product(bool, ReoptimizeCallSequences, true, DIAGNOSTIC,                  \
           "Reoptimize code-sequences of calls at runtime.")                 \
                                                                             \
   /* Power 8: Configure Data Stream Control Register. */                    \
-  product(uint64_t,DSCR_PPC64, (uintx)-1,                                   \
+  product(uint64_t, DSCR_PPC64, (uint64_t)-1,                               \
```

Comment:
Has nothing to do with this issue. But I leave it up to you.

Reply:
Just cleanup, no real change.